src/proto/net.proto

```diff
@@ -3,17 +3,31 @@ syntax = "proto2";
 package proto;
 
 message ServerMessage {
+  optional bool status = 1;
+
   oneof message_type {
```

Comment:
Hmm I'll keep it as-is for now.  Is `optional oneof` a thing though?  Couldn't find info on it.